src/components/shell/shell.js

```diff
@@ -219,12 +219,19 @@ class Shell extends Component {
    */
   componentDidMount() {
     const {player, eventManager} = this.props;
+    eventManager.listen(
+      document,
+      'scroll',
+      debounce(() => {
```

Comment:
remove the anonymous function and use arrow function for the callback `_onDocumentScroll`